lib/ansible/modules/database/postgresql/postgresql_db.py

```diff
@@ -391,6 +391,10 @@ def db_restore(module, target, target_opts="",
     elif os.path.splitext(target)[-1] == '.tar':
         flags.append(' --format=Tar')
         cmd = module.get_bin_path('pg_restore', True)
+        
```

Comment:
Please remove the trailing whitespace here:
`blank line contains whitespace`

Reply:
Fixed